### ISSUE TITLE
Add adbility for tokens to be used to create alt text

### ISSUE THREAD

**Issue by albanycomputers** [BWPanda/pdf_to_image#10]
The module doesn't seem to allow ALT text to be added to the images generated. When used on WCAG-compliant sites, the module fails to meet accessibility guidelines due to the lack of ALT text.

Add the facility to use TOKENS to generate the ALT text.

**Comment by a8w4** [comment]
Hi Steve,

thank you for your interest in the module and for your pull request. I'll review it as soon as I have time.

Kind regards,
Alexander
